<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TNG</class>
 <widget class="QMainWindow" name="TNG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>836</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>75</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="VDC">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>140</y>
      <width>116</width>
      <height>56</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>95</y>
      <width>316</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Molot</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Напряжение, приложенное к диоду</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="PlotWidget" name="currGraph" native="true">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>70</y>
      <width>321</width>
      <height>166</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="pulseGraph" native="true">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>290</y>
      <width>321</width>
      <height>186</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>240</y>
      <width>326</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Molot</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Импульсы , генерируемые диодом</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>155</y>
      <width>31</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Molot</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>B</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>344</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comList"/>
     </item>
     <item>
      <widget class="QPushButton" name="openCom">
       <property name="text">
        <string>Открыть порт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeCom">
       <property name="text">
        <string>Закрыть порт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refrCom">
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLCDNumber" name="Curr">
    <property name="geometry">
     <rect>
      <x>585</x>
      <y>33</y>
      <width>86</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>406</x>
      <y>40</y>
      <width>152</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Molot</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ток в цепи диода</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>685</x>
      <y>40</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Molot</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>35</y>
      <width>16</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Molot</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>m</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>681</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>215</y>
      <width>316</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Molot</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Датчик температуры</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>525</y>
      <width>316</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Molot</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Калибровка диода</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="thresSlider">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>325</y>
      <width>256</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="value">
     <number>22</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>265</y>
      <width>316</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Molot</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Уровень срабатывания</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="thresHold">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>270</y>
      <width>66</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>215</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>270</y>
      <width>31</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Molot</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>°С</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>365</y>
      <width>202</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Molot</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>температура достигнута</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>410</y>
      <width>224</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Molot</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>температура выше уровня</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>455</y>
      <width>220</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Molot</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>температура ниже уровня</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="imgTempOk">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>360</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>405</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>450</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
